lib/worker/base.js

```diff
@@ -176,7 +176,21 @@ const run = async options => {
 
 	try {
 		for await (const ref of (options.require || [])) {
-			await load(ref);
+			if (typeof ref === 'string') {
+				await load(ref);
+			} else if (Array.isArray(ref)) {
+				const [path, options] = ref;
+
+				const loadedModule = await load(path);
+
+				if (typeof loadedModule === 'function') {
+					loadedModule.apply(null, ...options);
```

Comment:
Let's `await` this too, so it can do some async setup before we load the next module.

And below for `default.apply`.

Do we care about the `this` argument? Could `loadedModule(...options)` instead.